aten/src/ATen/cuda/Sleep.cu

```diff
@@ -32,4 +33,37 @@ void sleep(int64_t cycles) {
   C10_CUDA_KERNEL_LAUNCH_CHECK();
 }
 
+#ifdef USE_ROCM
+__global__ void flush_icache_kernel()
```

Comment:
Can this be hidden as a API inside hipBLASLt?

Reply:
No, it can't.  The TunableOp interface needs to call it before each rocblas or hipblaslt kernel when icache flush is enabled.  I thought putting it into Sleep.cu would be ok.  Sleep.cu is a small utility kernel used by unit tests.  The ROCm-only flush_icache_kernel is a small utility kernel.  I didn't want to put it into its own *.cu file under aten/src/ATen/cuda/tunable because that directory is free from any kernels and it was cleaner to keep it that way.